cmd/helm/install.go

```diff
@@ -285,12 +286,32 @@ func (v *values) Set(data string) error {
 	return nil
 }
 
+func typedVal(val string) interface{} {
+	if val == "true" {
+		return true
+	}
+
+	if val == "false" {
+		return false
+	}
+
+	if iv, err := strconv.ParseInt(val, 10, 64); err == nil {
```

Comment:
The YAML converter seemed to quote these, which then forced them to be interpreted as strings. So I think we need this in order to avoid the issue people are seeing with the strict schema enforcement in Kube 1.4.